ci-operator/config/openshift-priv/cloud-network-config-controller/openshift-priv-cloud-network-config-controller-release-4.11.yaml

```diff
@@ -58,6 +58,14 @@ tests:
   commands: make test
   container:
     from: src
+- as: e2e-aws-serial
+  steps:
+    cluster_profile: aws
+    workflow: openshift-e2e-aws-serial
+- as: e2e-aws-ovn-serial
```

Comment:
you want to make this optional:true?
this job is not in a good state and I don't think we want to make it a required job

Reply:
It's the only test suite that we have though for egressip and the cncc's only purpose is cncc. Alternatively, we can make it optional but then in the cncc, ppl should run that test manually at least once to make sure that egressip passes. With that in mind, do you still want to make it optional (I have no opinion, here :) )

Reply:
Ok I'm going to settle for the more conservative approach and make them `optional: true` for the time being :-)

Reply:
I don't watch CNO PRs too much, but with aws-ovn-serial job passing at < 50% (recently it was much worse) the CNO
approvers would be doing a lot of /override to get PRs merged.